content/docs/README.md

```diff
@@ -30,6 +30,8 @@ $ kubectl kudo init --dry-run -o=yaml > kudo.yaml
 $ kubectl apply -f kudo.yaml
 ```
```

Comment:
I'd remove this example and its introductory sentence, and instead mention the `kubectl apply -f` in the bullet point which documents `--dry-run` in `cli.md`. But I know this is out of scope of this PR really.